miso-web/src/main/webapp/scripts/hot.js

```diff
@@ -1299,27 +1327,27 @@ var HotUtils = {
     setReadOnly(design || (template && template.idProperty));
   },
 
-  showDialogForBoxCreation: function(title, okButton, fields, pageURL, generateParams, getItemCount){
+  showDialogForBoxCreation: function(title, okButton, fields, pageURL, generateParams, getItemCount) {
     fields.push(ListUtils.createBoxField);
     Utils.showDialog(title, okButton, fields, function(result) {
       var params = generateParams(result);
-      if(params == null){
+      if (params == null) {
         return;
       }
-      var loadPage = function(){
+      var loadPage = function() {
         window.location = window.location.origin + pageURL + jQuery.param(params);
       }
-      if (result.createBox){
-        Utils.createBoxDialog(result, getItemCount, function(newBox){
+      if (result.createBox) {
+        Utils.createBoxDialog(result, getItemCount, function(newBox) {
           params.boxId = newBox.id;
           loadPage();
         });
       } else {
         loadPage();
       }
-    }); 
+    });
   }
-  
+
 };
 
 HotTarget = {};
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "HotTarget" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)